res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1712,6 +1940,10 @@ PioneerDDJSX.samplerLedsPlay = function(value, group, control) {
         sampleDeck = "[Sampler" + (samplerIndex + 1) + "]",
         padNum = PioneerDDJSX.samplerGroups[sampleDeck];
 
+    if (!engine.getValue(sampleDeck, "duration")) {
```

Comment:
I had the case where no track was loaded or a track was ejected and the `track_loaded` control was `true`, but I didn't test it in every detail.